app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
-
-### How to control your cookies
-
-You can delete or manage all cookies from your browser. The links below
-take you to the ‘Help' sections for each of the major browsers, so that
-you can find out more about how to manage your cookies.
-
-- [**Google Chrome**](https://support.google.com/chrome/answer/95647?hl=en-GB&p=cpn_cookies)
-- [**Microsoft Edge**](https://support.microsoft.com/en-us/help/4468242/microsoft-edge-browsing-data-and-privacy)
-- [**Microsoft Internet Explorer**](https://support.microsoft.com/en-us/help/17442/windows-internet-explorer-delete-manage-cookies#ie=ie-11)
-- [**Mozilla Firefox**](https://support.mozilla.org/en-US/kb/cookies-information-websites-store-on-your-computer?redirectlocale=en-US&redirectslug=Cookies)
-- [**Safari**](https://support.apple.com/kb/ph21411?locale=en_GB)
+| `_pin_unauth`        | Used by Pinterest to collect information about how their visitors interact with the pages on the website to track the efficiency                                      | 1 year                 |
+| `sc_at`              | Used by Snapchat to deliver more relevant and meaningful adverts. Also used to track the efficiency                                                                   | 12 months              |
+| `_scid`              | Used by Snapchat for tracking your actions, such as selecting recommended adverts. This information is used to make adverts more meaningful and relevant to you       | 12 months              |
+| `_gac_UA-179982899-1`| Records how you’ve interacted with advertising campaigns                                                                                                              | 90 days                |
+| `_gac_gb_76JLPNZBCP` | Used by Google Analytics and Google Ads to help us understand how you've interacted with advertising campaigns                                                        | 90 days                |
+| `_gcl_au`            | Used by Google Analytics and Google Ads to help us understand how you've interacted with advertising campaigns                                                        | 90 days                |
+| `_fbp`               | Used by Facebook to store and track your visits across websites                                                                                                       | 90 days                |
+| `_fbc`               | Used by Facebook to record that you’ve accessed our website using a Facebook advert                                                                                   | 90 days                |
+
+## Your choices regarding cookies
+
+If you prefer to avoid the use of cookies on the website, you must disable the use of cookies in your browser and then delete the cookies saved in your browser associated with this website. You may use this option for preventing the use of cookies at any time.
```

Comment:
```suggestion
If you prefer to avoid the use of Cookies on the Website, you must disable the use of Cookies in your browser and then delete the Cookies saved in your browser associated with this Website. You may use this option for preventing the use of Cookies at any time.
```